__tests__/endpoint.test.js

```diff
@@ -332,3 +332,85 @@ describe('POST/api/articles/:article_id/comment', () => {
   });
 
 })
+
+describe('PATCH/api/articles/:article_id', () => {
+
+  test('PATCH:200 sends a single article object to the client with vote increment', () => {
+
+      const testVote = { vote_increment : 6 }
+
+    return request(app)
+      .patch("/api/articles/1").send(testVote)
+      .expect(200)
+      .then((response) => {
+
+        const articleObj= response.body
+      
+        expect(articleObj).toBeInstanceOf(Object)
+      
+        articleObj.article.forEach((obj)=>{
+          expect(obj).toHaveProperty("article_id", expect.any(Number))
+          expect(obj).toHaveProperty("author", expect.any(String))
+          expect(obj).toHaveProperty("title", expect.any(String))
+          expect(obj).toHaveProperty("topic", expect.any(String))
+          expect(obj).toHaveProperty("created_at", expect.any(String))
+          expect(obj).toHaveProperty("article_img_url", expect.any(String))
+          expect(obj).toHaveProperty("body", expect.any(String))
+          expect(obj).toHaveProperty("votes")
+          expect(obj.votes).toBe(106) 
+      });
+    })
+  });  
+  test('PATCH:400 sends a error when invalid id is passed', () => {
+
+      const testVote = { vote_increment : 6 }
+
+    return request(app)
+      .patch("/api/articles/not-a-id").send(testVote)
+      .expect(400)
+      .then((response) => {
+
+        expect(response.body.msg).toBe('Bad request');
+        
+      });
+    })
+    test('PATCH:400 sends a error when vote_increment value is not a number', () => {
+
+      const testVote = { vote_increment : 'not-a-number' }
+
+    return request(app)
+      .patch("/api/articles/6").send(testVote)
+      .expect(400)
+      .then((response) => {
+
+        expect(response.body.msg).toBe('Bad request');
+        
+      });
+    })
+    test('PATCH:400 sends a error when object key is not vote_increment is incorrect', () => {
+
+      const testVote = { 789 : 9 }
```

Comment:
I believe we cannot use numbers as column_names in PSQL, so I would consider changing 789 here to a string to make sure we are testing having the wrong key and not the wrong data type